trie/stacktrie.go

```diff
@@ -381,8 +384,8 @@ func (st *StackTrie) Hash() (h common.Hash) {
 	st.hash()
 	if len(st.val) != 32 {
 		// If the node's RLP isn't 32 bytes long, the node will not
-		// be hashed, and instead contain the  rlp-encoding of the
-		// node. For the top level node, we need to force the hashing.
+		// be committed, and instead contain the  rlp-encoding of the
```

Comment:
The fix is in the wrong place :P 

For hasher, it's `hased`, `hashing`. We should put the fix in the commit function.

Reply:
doh!

Reply:
fixed